content/docs/1.8.2/important-notes/_index.md

```diff
@@ -41,6 +43,24 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
 
 The functionality of the [environment check script](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) (`environment_check.sh`) overlaps with that of the Longhorn CLI, which is available starting with v1.7.0. Because of this, the script is deprecated in v1.7.0 and is scheduled for removal in v1.9.0.
 
+## Breaking Change
+
+### V2 Backing Image
+
+Starting with Longhorn v1.8.2, V2 backing images are incompatible with earlier versions due to naming conflicts in the extended attributes (`xattrs`) used by SPDK backing image logical volumes. As a result, V2 backing images must be deleted and recreated during the upgrade process. Since backing images cannot be deleted while volumes using them still exist, you must first back up, delete, and later restore those volumes as the following steps:
+
+- Before upgrading to v1.8.2:
```

Comment:
I am curious. Is it possible for users to detach volumes, recreate the backing image, and then reattach volumes with the recrated BI without backups? Should it work as well?